basis/common-dto/src/main/java/com/alexkoveckiy/common/dto/configuration/CommonDtoConfig.java

```diff
@@ -0,0 +1,13 @@
+package com.alexkoveckiy.common.dto.configuration;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Created by alex on 01.03.17.
+ */
+
+@Configuration
+@ComponentScan(basePackages = "com.alexkoveckiy.common.dto")
```

Comment:
@Alexey007gold Ok. But still What you suppose to start by your configuration in this module ?